features/old/api/features.feature

```diff
@@ -8,59 +8,60 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
```

Comment:
I think this is not needed here. This does not affect the functionality of this specific scenario, as `I go to the edit page for plan "Basic"` will pick the **first** plan, which in this case will be the application plan